proposals/i18n-system-languages.md

```diff
@@ -0,0 +1,97 @@
+# Proposal: i18n.getPreferredSystemLanguages() and i18n.getSystemUILanguage()
+
+**Summary**
+
+Allows getting the language(s)/locale(s) of your operating system as a [BCP47 language tag](https://www.rfc-editor.org/bcp/bcp47.html)
+
+**Document Metadata**
+
+**Author:** [carlosjeurissen](https://github.com/carlosjeurissen)
+
+**Sponsoring Browser:** TBD
+
+**Created:** 2024-03-18
+
+**Related Issues:**
+https://github.com/w3c/webextensions/issues/252
+https://bugzilla.mozilla.org/show_bug.cgi?id=1888486
+
+## Motivation
+
+### Objective
+
+Allow extensions to display UI in the user's system language(s), even when this is distinct from the browser's UI language (which is restricted to languages supported by the browser).
+
+#### Use Cases
+
+Having the original system language(s) is useful for language-related extensions and for extensions who want to provide translations more true to the system language. For example, to adapt to a specific sub-language, like Belgium-Dutch (nl-BE). While i18n.getUILanguage would return 'nl'.
```

Comment:
@aphillips Thanks so much for the suggestions! Applied the suggested change.

Also added a better example of the purest form use-case which describes `i18n.getUILanguage` returning a language tag with additional subtags which might not be sufficient for the user.

Lastly rewritten sentences to use locale instead of language where it makes sense based on your suggestion.